package.json

```diff
@@ -50,6 +50,11 @@
   "main": "./out/extension.js",
   "contributes": {
     "commands": [
+      {
+        "command": "flatpak-vscode.show-output-channel",
+        "title": "Shows the output channel",
+        "category": "Flatpak"
+      },
```

Comment:
Does this command has to be part of the command palette? Just asking

Reply:
hmm, I think it makes sense to have it there, but I don't have strong opinion on either side

Reply:
let us keep it then, could be useful for people not very familiar with vscode

Reply:
It wasn't added though in command palette, only registered as command

Reply:
nvm, it seems it is also added in command palette